### ISSUE TITLE
Window size

### ISSUE THREAD

**Issue by davidgv** [eyalroz/removedupes#198]
The window where duplicates are listed seems very small, with all data very close together and in many cases illegible.
Maybe you could do one of these two options (or both):
- Possibility to resize the results window
- Include scroll bars in the results window, to be able to make the columns larger and scroll through them.

**Comment by eyalroz** [comment]
Indeed, the review dialog is kind of broken. To my defense I'll say that this is not my fault: TB broke the XUL flex property, which many of us were using for decades to form our dialogs properly, and now it's quite difficult to get sizes and proportions right. I should definitely work on that though.

**Comment by davidgv** [comment]
Thank you.

**Comment by reny77** [comment]
is there any news? it's really annoying

**Comment by bkmdev** [comment]
Any update on this?  or at least a workaround?  I too just found this useful add-on (thanks!) but was a bit disappointed when I found that one couldn't resize the window on larger displays 😞  Thanks!

PS:  Anything we can do to help?  Are column sizes determined by CSS rules?

**Comment by davidgv** [comment]
Yes, can we do anything to help?

**Comment by OnkelTom-76** [comment]
This is a great add-on, no doubt. Appreciate the work of the developer, but this is still persisting. Makes it kind of difficult to use. @eyalroz Would you mind, looking into this, once again? Thanks for your work and support.
